src/lib.rs

```diff
@@ -28,6 +28,7 @@ pub mod transports;
 pub mod types;
 
 pub use api::{Web3Main as Web3, ErasedWeb3};
+pub use helpers::CallResult;
```

Comment:
imo it helps with docs readability -> https://tomusdrw.github.io/rust-web3/web3/api/eth/struct.Eth.html

All functions return `CallResult` and to know what this structure is, I need to open the source code